roles/step_ca/README.md

```diff
@@ -148,21 +148,14 @@ This role will only decrypt the root key for as long as strictly neccessary.
 
 ---
 
-##### `step_ca_existing_<root/key>`
-- Whether to use an existing root certificate/key and if so from where to import it from
-- Choices:
-    - `remote`: The root certificate/key is already present on the remote host
-    - `local`: The root certificate/key is read from the controller
-- Note that both cert and key need to be either imported, **or** generated.
-  For example, you cannot import the key but generate the certificate
-- Default: Not set.
-    - If unset and `_root/key_file` is also unset, a new certificate will be generated
-    - If unset and `_root/key_file` is set, the files are treated as `remote` to preserve backwards-compatibility to previous collection versions.
-      This behavior may be removed in a future release
-
 ##### `step_ca_existing_<root/key>_file`
 - The path of an existing PEM file to be used as the root certificate/key
-- Depending on the value of `step_ca_existing_<root/key>`, the file must either be on the remote host or the controller
+- If the file is present on the controller instead of the target node, set `step_ca_existing_<root/key>_is_local`, to `true`.
+- Default: not set (will generate a new certificate)
+
+##### `step_ca_existing_<root/key>_is_local`
```

Comment:
Instead of `is_local`, perhaps a `_mode` parameter would be better, with the default behavior of `none` and choices for `local` and `remote`. This would allow for easy extendability in the future